WordPress/UITests/Tests/EditorGutenbergTests.swift

```diff
@@ -7,8 +7,8 @@ class EditorGutenbergTests: XCTestCase {
     override func setUpWithError() throws {
         setUpTestSuite()
 
-        _ = try LoginFlow.login(siteUrl: WPUITestCredentials.testWPcomSiteAddress, email: WPUITestCredentials.testWPcomUserEmail, password: WPUITestCredentials.testWPcomPassword)
-        editorScreen = try EditorFlow
```

Comment:
With the test no longer using the stored `editorScreen`, we should be able to remove the property from the class, too. See line 5 of this file.